mongodb_consistent_backup/Upload/S3/S3.py

```diff
@@ -1,6 +1,7 @@
 import os
 import logging
 
+import boto
```

Comment:
PEP8 says to put imports from the standard library at the top, and imports from third party packages underneath as islue has done.